server/views/pages/overview/partials/overviewTab/overviewTabContents.njk

```diff
@@ -8,6 +8,17 @@
   </div>
 
   <div class="govuk-grid-column-one-third app-u-print-full-width">
+    {{ actionsCard({
+      actions: [
+        { title: 'Add a new goal', href: '/plan/' + prisonNumber + '/goals/create', id: 'add-goal-button', 'render-if': hasEditAuthority },
```

Comment:
The first action item (Add a new goal) should only be shown if the user has the edit authority - hence the need for the `render-if` attribute